testhelper/integrationtesthelper.go

```diff
@@ -35,29 +44,115 @@ const (
 	NumIMUData = 40
 	// Path to slam mock data used for integration tests artifact path.
 	// artifact.MustPath("viam-cartographer/mock_lidar").
-	mockDataPath                = "viam-cartographer/mock_data"
-	sensorDataIngestionWaitTime = 50 * time.Millisecond
+	mockDataPath                      = "viam-cartographer/mock_data"
+	sensorDataIngestionWaitTime       = 50 * time.Millisecond
+	defaultLidarTimeInterval          = 200
+	defaultMovementSensorTimeInterval = 50
+	testTimeout                       = 20 * time.Second
 )
 
 var defaultTime = time.Time{}
 
-// TimeTracker stores the current and next timestamps for both IMU and lidar. These are used to manually
+// Test final position and orientation are at approximately the expected values.
+func testCartographerPosition(t *testing.T, svc slam.Service, useIMU bool, expectedComponentRef string) {
```

Comment:
This function and the functions below were moved from `integration_test.go`. The expected values in this function had to be changed since a bug in the integration tests was fixed in this PR.